### ISSUE TITLE
ffmpeg fails to create m4b file with embedded cover art

### ISSUE THREAD

**Issue by Diegus83** [ping/odmpy#6]
When trying to create a merged m4b file with the options
`odmpy -v dl -c -m --mergeformat m4b -k -f file.odm`

ffmpeg fails because it can't use the embedded cover art
```
[mp4 @ 0x7fc62a802e00] Could not find tag for codec h264 in stream #0, codec not currently supported in container
Could not write header for output file #0 (incorrect codec parameters ?): Invalid argument
Error initializing output stream 0:1 -- 
[aac @ 0x7fc62a805e00] Qavg: nan
Conversion failed!
ffmpeg exited with the code: 1
```

ffmpeg -version output
```
ffmpeg version 4.3.1 Copyright (c) 2000-2020 the FFmpeg developers
built with Apple clang version 11.0.0 (clang-1100.0.33.17)
```

**Comment by ping** [comment]
I've tried to fix this without a new dependency as required in your PR #7. Can you check if the latest master resolves this for you? I tested it with a couple of players (VLC, IINA on MacOS and Smart AudioBook Player, Voice on Android) and the cover art does show up.

**Comment by Diegus83** [comment]
Tried two different titles and it is working! It shows up in Prologue and Plex as well. Thanks!

Interesting, I tried the `-disposition:v:0', 'attached_pic'` option before in an m4b file and the resulting file had the artwork playing as a video track in VLC. I was either mapping the tracks wrong or adding artwork to an already encoded mp4 is different from adding during the encoding.

I read that embedded artwork in mp4 files is sort of hack and not really part of the specification so maybe that's why it is so hit and miss.
